compiler/rustc_typeck/src/collect.rs

```diff
@@ -2387,7 +2387,7 @@ fn compute_sig_of_foreign_fn_decl<'tcx>(
                     .sess
                     .source_map()
                     .span_to_snippet(ast_ty.span)
-                    .map_or(String::new(), |s| format!(" `{}`", s));
+                    .map_or_else(|_| String::new(), |s| format!(" `{}`", s));
```

Comment:
```suggestion
                    .map_or_else(String::new, |s| format!(" `{}`", s));
```

Reply:
This was marked as resolved but it's not resolved.

Reply:
Ah, this is `Result`, not `Option`, nvm.